src/main/java/seedu/ccacommander/logic/commands/EditEventCommand.java

```diff
@@ -83,7 +86,11 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setEvent(eventToEdit, editedEvent);
         model.updateFilteredEventList(PREDICATE_SHOW_ALL_EVENTS);
+        model.updateFilteredMemberList(PREDICATE_SHOW_ALL_MEMBERS);
```

Comment:
So that in case the comment is called after a viewEvent, the hours and remark field will disappear

Reply:
Because leaving unupdated can cause these troubles for now:
this sequence of actions
viewMember 1
editEvent 1 l/SH
this will set the hours and remark of event 1 back to 0 and blank; and after clicking the card, the two fields will disappear.
Updating the filtered list avoid such UI bugs